lib/vet360/models/address.rb

```diff
@@ -28,6 +36,50 @@ class Address < Base
       attribute :zip_code_suffix, String
 
       validates :source_date, presence: true
+
+      validates(
+        :address_pou,
+        presence: true,
+        inclusion: { in: ADDRESS_POUS }
+      )
+
+      validates(
+        :address_type,
+        presence: true,
+        inclusion: { in: ADDRESS_TYPES }
+      )
+
+      def details
+        case address_type
+        when DOMESTIC
+          domestic_address
+        when INTERNATIONAL
+          international_address
+        when MILITARY
+          military_address
+        end
+      end
+
+      def domestic_address
+        {
+          id: id,
+          address_line_1: address_line_1,
+          address_line_2: address_line_2,
+          address_line_3: address_line_3,
+          city: city,
+          state_abbr: state_abbr,
+          zip_code: zip_code,
+          zip_code_suffix: zip_code_suffix
+        }
+      end
+
+      def international_address
+        # TODO
```

Comment:
@rlbaker @emersonthis - These two methods, and the constants I chose above, are placeholdes until we get answers and closure on our discussion around the criteria for the different address types